Algo-Workout/AW-T3
# #9 - OAuth-Github Discovery

## Overview

**Issue Type**


**Description**
- Githuh Oauth successfully logs in with github user info and session
- Added separate button for Github Auth
- Added separate icons for Discord and Github login functionality

**Steps to Validate Feature ** 
1. Run `npm run dev`
2. Click on `Sign into this application with Github!`
3. Proceed to login with GitHub user credentials
4. Open network tab and double check for session-token
5. Log out after complete

**Attachments and other context**
![github Oauth demo](https://github.com/Algo-Workout/AW-T3/assets/112226540/d08a5cc9-5ba5-4c7b-a675-97b15bc22738)
![Screenshot 2023-08-31 at 2 21 26 PM](https://github.com/Algo-Workout/AW-T3/assets/112226540/3291b396-f58c-4669-aa0a-4ab96f004851)
